src/apps/persistence/workspace.clj

```diff
@@ -24,16 +26,34 @@
        (map-indexed (partial app-groups/add-subgroup root-category-id))
        (dorun)))
 
+(defn set-workspace-root-app-group
```

Comment:
Good catch on this and the dead code.
Did you run a linter to find these? My `lein eastwood` linter didn't pick these up.